cogs/auto_standup.py

```diff
@@ -0,0 +1,45 @@
+import discord
+import asyncio
+import datetime
+from datetime import timezone
+from discord import app_commands
+from discord.ext import commands
+
+channel_ID = 1172727647382552737
+day_seconds = 86400
```

Comment:
Is `day_seconds` in relation to the number of seconds in a week? If so, make this should be changed to another variable name like `week_as_seconds` to make it clearer